llvm/lib/SYCLLowerIR/ESIMD/LowerESIMD.cpp

```diff
@@ -1271,6 +1271,20 @@ translateSpirvGlobalUses(LoadInst *LI, StringRef SpirvGlobalName,
   }
 }
 
+static void translateGlobalUse(Value *Use, StringRef SpirvGlobalName,
```

Comment:
just wondering, how did this issue come up? do we have an internal tracker for it?